app/display/representation-javafx/src/main/java/org/csstudio/display/builder/representation/javafx/SVGHelper.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (C) 2019 European Spallation Source ERIC.
+ *
+ *  This program is free software; you can redistribute it and/or
+ *  modify it under the terms of the GNU General Public License
+ *  as published by the Free Software Foundation; either version 2
+ *  of the License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program; if not, write to the Free Software
+ *  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
+ *
+ */
+
+package org.csstudio.display.builder.representation.javafx;
+
+import javafx.embed.swing.SwingFXUtils;
+import javafx.scene.image.Image;
+import org.apache.batik.transcoder.TranscoderException;
+import org.apache.batik.transcoder.TranscoderInput;
+import org.apache.batik.transcoder.TranscoderOutput;
+import org.apache.batik.transcoder.image.ImageTranscoder;
+
+import java.awt.image.BufferedImage;
+import java.io.InputStream;
+import java.text.MessageFormat;
+
+/**
+ * Helper class to load SVG files. It is based on the following post on Stackoverflow:
+ * https://stackoverflow.com/questions/12436274/svg-image-in-javafx-2-2
+ *
+ * The dependency to Apache Batik inflates the size of the lib folder by ~3 MB.
+ *
+ * Testing shows that gradients may not always render correctly. Maybe a Batik issue?
+ */
+public class SVGHelper {
+
+    private static BufferedImageTranscoder bufferedImageTranscoder;
+
+    static{
+        bufferedImageTranscoder = new BufferedImageTranscoder();
+    }
+
+    /**
+     * Loads SVG file as an {@link Image}.
+     * @param svgFilePath Non-null input stream to SVG file.
+     * @return A {@link Image} object if all goes well.
+     */
+    public static Image loadSVG(InputStream svgFilePath) {
+        TranscoderInput input = new TranscoderInput(svgFilePath);
+        try {
+            bufferedImageTranscoder.transcode(input, null);
+            return SwingFXUtils.toFXImage(bufferedImageTranscoder.getBufferedImage(), null);
+        } catch (TranscoderException e) {
+            throw new RuntimeException(MessageFormat.format("Unable to transcode SVG file, cause: {0}", e.getMessage()));
```

Comment:
e.getMessage() might be empty, plus the original stack trace of the TranscoderException is lost.
Suggest `throw new RuntimeException("Unable to transcode SVG file", e)`.
Also, since calling code some levels up already catches Exception, throw plain Exception instead of RuntimeException.